infra/modules/aws-psoxy-lambda/main.tf

```diff
@@ -44,7 +44,7 @@ resource "aws_lambda_function" "instance" {
   function_name                  = local.function_name
   role                           = aws_iam_role.iam_for_lambda.arn
   architectures                  = ["arm64"] # 20% cheaper per ms exec time than x86_64
-  runtime                        = "java11"
+  runtime                        = "java17"
```

Comment:
BTW; worth to parametrize this ? By default `java17`, but it can be customized with any other value depending on the needs

Reply:
YAGNI ... I don't see utility of it. java11 is EoL, deprecated by Oracle since Sept 2023; will be deprecated by GCP as a runtime in Sept 2024.  

if someone asks for running in java11 jre, then we can do it  - but I just don't what the use-case would be. If anything, I think they'd want opposite (forcing to java17 or java21, as those are more fully supported)